cilium-health/launch/endpoint.go

```diff
@@ -345,18 +344,6 @@ func LaunchAsEndpoint(baseCtx context.Context,
 		return nil, fmt.Errorf("Error while configuring routes: %s", err)
 	}
 
-	if option.Config.IPAM == ipamOption.IPAMENI {
```

Comment:
If you're removing these bits, then the `routingConfig` now becomes unused and can be removed.

(Though I'll also naturally want to ask, if this code is not handled here then where is it handled? Maybe I'll figure that out later in the review..)

Reply:
btw, here be dragons: We identified and manually tested the fix that introduced this code but I don't think there's any automated testing or regression checking.

Reply:
Thanks for the heads-up. `routingConfig` removed, and yes, it seems that we have to rely on manual testing here.